release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -477,6 +477,40 @@ Available as a Technology Preview, the Insights Runtime Extractor feature gather
 Red{nbsp}Hat uses the collected runtime workload data to gain insights that can help you make investment decisions that will drive and optimize how you use your {product-title} containers.
 For more information, see xref:../nodes/clusters/nodes-cluster-enabling-features.adoc#nodes-cluster-enabling[_Enabling features using feature gates_].
 
+[id="ocp-release-notes-insights-operator-rapid-recommendations_{context}"]
+==== Rapid Recommendations
+
+In this release, enhancements have been made to the Rapid Recommendations mechanism for remotely configuring the rules that determine the data that the Insights Operator collects.
```

Comment:
```suggestion
In this release, the rapid recommendations feature is enhanced by becoming version-independent, and builds on the existing conditional data gathering mechanism. The Insights Operator connects to a secure remote endpoint service running on [/console.redhat.com](link:https://console.redhat.com[console.redhat.com]) to retrieve definitions that contain the rules for determining which container log messages are filtered and collected by Red{nbsp}Hat.
```

Reply:
This is technically inaccurate. The feature was version-independent in the previous release but now further under-cover enhancements have been made for performance and scaling etc...
As this feature is still quite new and key to the Insights Operator evolution, the PM and engineering team want us to reiterate what we had for 4.17 as they have significantly enhanced the feature under the covers.

Reply:
No worries, my general suggestion is based on minimalism. I was trying to find ways to trim some of the text. I leave technical accuracy to you.